### PR TITLE
quick steps to contribution to this project

### PR DESCRIPTION
**Pull request by tomtranschool**
This is in relation to #1

### CODE REVIEW

**Comment by tomtranschool** [comment]
@nwoodsGIA

**Comment by tomtranschool** [comment]
@nwoodsgia

**Comment by tomtranschool** [comment]
:poop:

**Comment by nwoodsgia** [comment]
looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # My First Project
 This is a practice repository for learning Github in the Web Interface.
+
+## How to Contribute 
+Steps for contributing can be found in the [CONTRIBUTING.md](/CONTRIBUTING.md) file.
```
Comment (tomtranschool) [inline README.md:5]: Awesome use of Terminology in GITHUB

**Comment by nwoodsgia** [comment]
: )

**Pull request merged**